cmd/neofs-lens/internal/meta/status.go

```diff
@@ -60,6 +60,12 @@ func statusFunc(cmd *cobra.Command, _ []string) error {
 		cmd.Printf("\tBucket: %d\n"+
 			"\tValue (HEX): %s\n", bucket.BucketIndex, valStr)
 	}
+	if len(res.HeaderIndex) > 0 {
+		cmd.Println("Header field indexes:")
+		for _, field := range res.HeaderIndex {
+			cmd.Printf("\t%s: %x\n", field.K, field.V)
```

Comment:
`field.K` is a UTF-8 w/o zeros, it may be not human-readable

Reply:
what do you suggest here? it will always have human-readable system fields, and will likely have readable user attrs (why uft8 but not readable?). if we are afraid that it may not be readable, then it will not be readable for sure if we encode it somehow and reading it (first of all it is a debug util that is supposed to help with smth) will be harder

Reply:
ok, print only system attrs

Reply:
i just noticed that. In general, not all attributes can be read, copy-pasted, etc. But i checked that, for example, `neofs-cli object get`, also prints them as is (`%s`)

so it makes no sense to behave differently in this case only. Lets go back to the original version of ur code

Reply:
There is `%q`.